app/models/carto/user_migration_import.rb

```
@@ -55,6 +57,14 @@ def enqueue
 
     private
 
+    def assert_organization_does_not_exist
+      raise OrganizationAlreadyExists.new if organization.present? && Carto::Organization.exists?(organization.id)
+    end
+
+    def assert_user_does_not_exist
+      raise UserAlreadyExists.new if user.present && Carto::User.exists?(user.id)
+    end
```

I insist that these checks are incorrect, and I think the fact you needed to stub them in the tests is the proof. Let's talk and I'll explain in person.

---

I need to stub the test because I'm using fake data with the users and if they get to hit the db an error will raise